setup.py

```diff
@@ -72,16 +99,52 @@ def pkgconfig(*packages, **kw):
 
 if __name__ == '__main__':
 
-    if sys.platform == 'win32':
+    ipoptdir = os.environ.get('IPOPTWINDIR', '')
+
+    # conda forge hosts a windows version of ipopt for ipopt versions >= 3.13.
+    # The location of the headers and binaries are in $CONDA_PREFIX/Library/
+    # and the library binary is named "libipopt.lib". If the IPOPTWINDIR
+    # environment variable is set to USECONDAFORGEIPOPT then this setup will be
+    # run.
+    if sys.platform == 'win32' and ipoptdir == "USECONDAFORGEIPOPT":
```

Comment:
My proposed solution is to accept a very specific value for `IPOPTWINDIR`, which is set by the user to signal it is being build with the conda-forge windows binaries. I couldn't figure out a way to automatically detect this in a reliable way and could ensure that would be the users intent when building cyipopt. If anyone has a better idea, let me know.